addons/web/static/tests/views/fields/progress_bar_field_tests.js

```diff
@@ -594,7 +594,7 @@ QUnit.module("Fields", (hooks) => {
             const input = target.querySelector(".o_progressbar_value.o_input");
             assert.strictEqual(input.value, "99", "Initial value in input is correct");
 
-            await editInput(target, ".o_field_widget input", "1#037:9");
+            await editInput(target, ".o_field_widget input", "1037");
```

Comment:
This change defeats the point of the test. You should change the test to use `float_field` instead.

Reply:
> This change defeats the point of the test. You should change the test to use `float_field` instead.

this test `write float instead of int` meaning is changed if we use float_field field instead  float_integer. what do you think ?

Reply:
This test is checking that we can use both floats and integers, and most importantly floats with a custom locale. With this PR, we can no longer enter floats if the field is an integer, so the type of the field should be changed rather than using an integer instead of a float in the test.

Reply:
Now. I'v  updated the test title and  code. You can check it. thank you